lib/board/resources.c

```diff
@@ -1830,6 +1830,56 @@ static const uint8_t unplug_data[395] = {
 static const Image unplug_image = {45, 27, sizeof(unplug_data), unplug_data};
 const AnimationFrame unplug_frame = {208, 21, 20, 100, &unplug_image};
 
+
+/* --- General image data, 20x40 ---------------------------------------------- */
+static const uint8_t ethereum_icon_data[367] =
```

Comment:
What do you think about standardizing the icon bitmap size and optionally passing signed icon data to the device dynamically? With the dynamic strategy, we'd store the bitmap data in globally-accessible context and keep it available for the duration of  the signing interaction. We could keep a few default icons like this one in flash to fall back on in general cases.